npa.js

```diff
@@ -50,11 +49,16 @@ function npa (arg) {
   if (nameparse && validName(nameparse[3]) &&
       (!nameparse[2] || validName(nameparse[2]))) {
     res.name = (nameparse[1] || "") + nameparse[3]
-    if (nameparse[2])
+    if (nameparse[2]) {
       res.scope = "@" + nameparse[2]
+      res.escapedName = escapeName(res.name)
```

Comment:
Is there a reason to not just hoist this out of the conditional and just make it

```javascript
res.escapedName = escapeName(res.name)
```
?

Doing so lets us change the escaping function without tying it too heavily to scopes, and also removes the need to deal with `else` branches forever.